### ISSUE TITLE
typecheck.Error

### ISSUE THREAD

**Issue by michalstepniewski** [google/in-silico-labeling#4]
I am running the code on AWS r5a.xlarge machine with
  | Deep Learning AMI (Ubuntu) Version 14.0 - ami-0089d61bf6a518044-- | --

The machine has 32GB RAM and 4 vCPUs.

Running on 
tensorflow installed through conda 
(I guess python3.6 is used).
tensorflow.contrib.labeled_tensor.python.ops._typecheck.Error: ['batch', 'row', 'column', ('z', 0.0), ('channel', 'TRANSMISSION'), 'class'] of type <class 'list'> is not an instance of the allowed type Collection(Union((str,), Tuple((str,), collections.abc.Hashable))) for <function expand_dims at 0x7fa78c2bb840>

**Comment by emchristiansen** [comment]
What command are you executing?
And can you confirm the Python version?

**Comment by michalstepniewski** [comment]
Yes. I used python3.6. Anyways got it working now on pip installed version of TF. Excited to test it on my data :)
